etna/.github/workflows/r-tests.yml

```diff
@@ -0,0 +1,11 @@
+name: Run R tests
```

Comment:
If we add the R test stuff into the `release-test` and `test` subsections we shouldn't need this?  The base workflow runs the these actions automatically.  Although adding it here will cause the tests to also run in the etna repo itself.

Reply:
Cool, thanks, that's good to know. I think it's still good to add have the tests running in the separate repos (I ran into an issue last week where the CI results were different for Timur between `monoetna` and `timur`), so I'll try to get these working in `etna`, too.

Reply:
Do you know what the discrepancy is?  In general, if we identify those issues, we should try to get both repos to agree on test behavior.  I'd hate for, say, one of the repos to be running the tests wrong and misreport success / failure.  I can take a look if you have an example.

Reply:
Yeah, I think it was related to the issues you've run into with `npm link` and getting `etna-js` into Timur. There were dependencies in `etna-js` that were found / installed just fine in the docker container CI (`monoetna`), but weren't present in the standalone Timur repo and also thus not found during actual deployments, causing webpack to crash. I think it's related to [this code for etna install in the containers](https://github.com/mountetna/monoetna/blob/master/docker/etna-base/entrypoints/build.sh#L32)?

Reply:
https://github.com/mountetna/timur/runs/1397864450?check_suite_focus=true

Reply:
Ah shit, yea.  So this is a known bug in npm.  Basically, in normal circumstances, npm will cascade dependencies of dependencies on install.  _Except for linked packages_ which are bizarrely only installed in a shallow way.  (https://github.com/npm/npm/issues/17287#issuecomment-400833982) This is going to not be fixed for awhile, and the only solution is we must guarantee that all dependencies of etna-js are _also_ installed in those subprojects, until we finally move to full docker (in which case the linking will no longer be necessary).

Ack, figured this would bite us.